iterator.go

```diff
@@ -269,9 +209,7 @@ func (it *changeIterator[Obj]) refresh(txn ReadTxn) {
 	itxn := txn.getTxn()
 	indexEntry := itxn.root[it.table.tablePos()].indexes[RevisionIndexPos]
 	indexTxn := indexReadTxn{indexEntry.tree, indexEntry.unique}
-	updateIter := &iterator[Obj]{indexTxn.LowerBound(index.Uint64(it.revision + 1))}
-	deleteIter := it.dt.deleted(itxn, it.deleteRevision+1)
-	it.iter = NewDualIterator(deleteIter, updateIter)
+	it.iter = indexTxn.LowerBound(index.Uint64(it.revision + 1))
 
 	// It is enough to watch the revision index and not the graveyard since
```

Comment:
no more graveyard, so comment can go
generally probably grep for graveyard; there's some more comment references to it which are stale